ressources/LiberAbaci24/slides_LiberAbaci24.tex

```diff
@@ -0,0 +1,323 @@
+      \item<6-> It enables to \ul{centralise efforts} in a \ul{lasting way}
+      \item<7-> Being horizontal, it should be \ul{easier to maintain}
+      % \item<8-> It will \ul{showcase all that is possible} with Coq
+    \end{enumerate}
+  \end{tcbSol}
+\end{frame}
+
+\begin{frame}{Example: Documenting Equations}
+  \onslide<1->
+  \vspace*{-3pt}
+  \begin{tcbPbl}{The documentation was inadequate}
+    \begin{enumerate}
+      \item Many features: basics, dependent matching, wf recursion, ...
+      \item Despite many examples, the doc was not very fitted nor complete
+    \end{enumerate}
+  \end{tcbPbl}
+  \onslide<2->
+  \vspace*{-3pt}
+   \begin{tcbProp}{A new documentation}
+    \begin{enumerate}
+      \item We wrote 4 tutorials out of 6 planned, ~600l code/~1200l text
+      \item It is now much clear, and easy to improve or extend
+      \item We wrote it modularly with help from the community
+    \end{enumerate}
+  \end{tcbProp}
+  \only<2>{
+  \vspace*{-15pt}
+  \begin{figure}[H] \hspace*{-0.9cm}
+    \includegraphics[scale=0.35]{equations_tuto.jpg}
+  \end{figure}}
+  \only<3>{
+  \vspace*{-3pt}
+  \begin{tcbSol}{Many Improvements}
+  \begin{enumerate}
+    \item We found 2/3 bugs writing the documentation
+    \item We improved the behavior of the main tactic
+  \end{enumerate}
+  \end{tcbSol}}
+\end{frame}
+
+\subsection{Demonstration}
+
+% 3 - 5 minutes
+
+\begin{frame}
+  \begin{center}
+    \large
+    Search for \textbf{Coq Platform Docs}, or go on: \\
+    \textcolor{blue}{\url{https://coq.inria.fr/platform-docs/}}
+  \end{center}
+\end{frame}
+
+
+\subsection{Contributing}
+
+% \subsection{Contributing to the documentation}
+
+\begin{frame}{Contributing}
+  \begin{tcbObj}{Everyone can help}
+    \begin{enumerate}
+      \item<1-> \ul{Everyone can contribute} whatever their expertise !
+      \item<2-> \ul{Both} experts and casual users' opinions are valuable !
+    \end{enumerate}
+  \end{tcbObj}
+  \onslide<3->
+  \begin{tcbProp}{Contributing}
+    There are \ul{many ways} to contribute:
+    \begin{enumerate}
+      \item<3-> Give feedbacks on existing tutoand how-to, or wishes on
+      \textcolor{blue}{\href{https://www.theozimmermann.net/platform-docs/}{zulip}}
+      \item<4-> Answer questions, and share knowledge and folklore on
+      \textcolor{blue}{\href{https://www.theozimmermann.net/platform-docs/}{zulip}}
+      \item<5-> Write tutorials and how-to guides, or write PR to improve them
+      \item<6-> Help reviewing tutorials and how-to guides
+      \item<7-> Help setting up the repo, or the web-interface
+    \end{enumerate}
+  \end{tcbProp}
+\end{frame}
+
+\begin{frame}{Documenting Project and Plugins}
+  \begin{tcbProp}{Develop the documentation modularly}
+    For each projet or plugins, we would like to set up:
+    \begin{enumerate}
+      \item<2-> Set up a reviewer term with dev / maintainer / advanced users
+      \item<2-> Set up a project and open issues so that people can contribute
```

Comment:
```suggestion
    For each projet or plugins, we would like to set up:
    \begin{enumerate}
      \item<2-> a reviewer team with dev / maintainer / advanced users
      \item<2-> a project and open issues so that people can contribute
```

or:
```suggestion
    For each projet or plugins, we would like to:
    \begin{enumerate}
      \item<2-> Set up a reviewer team with dev / maintainer / advanced users
      \item<2-> Set up a project and open issues so that people can contribute
```